monitoring/grafana/grr_grafanalib_dashboards/util.py

```diff
@@ -0,0 +1,7 @@
+from grr_grafanalib_dashboards.config import DATA_SOURCE
+
+def add_data_source(dashboard):
```

Comment:
I think it would be cleaner to pass the data source as an argument. This way the utility function would be more narrow-scoped. You can import the DATA_SOURCE in a place where it's used and pass it as an argument.

Reply:
As of now, `DATA_SOURCE` is used nowhere else in the module; we can pass it as an argument from each `<grr_component>.dashboard.py` file, but then we would have to import `DATA_SOURCE` to there, achieving the same result.
What do you think?

Reply:
Yes, this is what I'd suggest. Then add_data_source would be a narrow-scoped utility function. Importing config.py in each dashboard also seems like a nice idea. Then for the reader of the dashboard code it'd be immediately clear where the config options are coming from.